### PR TITLE
Issue#49 add a configurable fixture template

### LINKED ISSUE #49: Task: Add a configurable fixture template
## Task Description

It would be nice to have a manifest which includes fixtures which are configurable  via `values.yaml`.

## Additional Information

* Use `{{ toYaml .Values.geonodeFixtures | nindent 2 }}` to include the fixtures.

Not sure about auto loading the fixtures via `python manage.py loaddata <yourfixturehere>`. For now this would have  be done manually. What is your opinion on this @mwallschlaeger ?

### PR DESCRIPTION
**Pull request by ridoo**
Closes #49

### CODE REVIEW

File: deployment/geonode/templates/_helpers.tpl [hunk 1]
```diff
@@ -37,7 +37,7 @@ amqp://{{ .Values.rabbitmq.auth.username }}:{{ .Values.rabbitmq.auth.password }}
 {{- end -}}
 
 {{- define "external_port" -}}
-{{- if or (eq (toString .Values.geonode.ingress.externalPort) "80") (eq (toString .Values.geonode.ingress.externalPort) "443") -}}
+{{- if or ( not .Values.geonode.ingress.externalPort ) (eq (toString .Values.geonode.ingress.externalPort) "80") (eq (toString .Values.geonode.ingress.externalPort) "443") -}}
```
Comment (ridoo) [inline deployment/geonode/templates/_helpers.tpl:40]: Urgh .. this should not be part of this PR .. I will remove this

File: deployment/geonode/values.yaml [hunk 2]
```diff
@@ -444,3 +444,8 @@ postgres-operator-ui:
...
...AAAAAAAAAAAAACYmJi4rKyukLCwt8jExTv83N3L/OjqE/zo6hP83N3L/MTFQ/ysrLPUrKyukKCgoMwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAZGRkKKCgoLCsrK4YsLCzNKysr9SsrK/YsLCzNLCwsiiYmJi4ZGRkKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==
```
Comment (mwallschlaeger) [inline deployment/geonode/values.yaml:451]: should this be something like:
```
favicon: |
```

**Review by mwallschlaeger — CHANGES_REQUESTED** [review]
As the values.yaml has changed please add documentation for the added parameters and run the doc generating tool https://github.com/norwoodj/helm-docs/releases/v1.11.0

**Review by mwallschlaeger — APPROVED** [review]

File: deployment/geonode/templates/geonode/geonode-env.yaml [hunk 3]
```diff
@@ -55,6 +55,7 @@ data:
 
   # General settings
   FREETEXT_KEYWORDS_READONLY: {{ include "boolean2str" .Values.geonode.general.freetext_keywords_readonly | quote }}
+  FIXTURE_DIRS: "[ '/usr/src/geonode/fixtures' ]"
```
Comment (ridoo) [inline deployment/geonode/templates/geonode/geonode-env.yaml:58]: @mwallschlaeger This does not match the actually configured volume:

https://github.com/zalf-rdm/geonode-k8s/pull/50/files#diff-fca9f77d2a6cb907ee65a0f6252b1261757188e043aed583e778fccdc9b1d4baR131

I will have to provide a quick fix for that

**Pull request merged**